stable/superset/README.md

```diff
@@ -50,6 +50,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `initFile`                 | Content of init shell script                    | See [values.yaml](./values.yaml)                             |
 | `replicas`                 | Number of replicas of superset                  | `1`                                                          |
 | `extraEnv`                 | Extra environment variables passed to pods      | `{}`                                                         |
+| `extraArguments`           | Extra arguments passed to init_superset.sh      | `[]`                                                         |
```

Comment:
In making this change, you need to provide an example of `extraArguments` passed to `init_superset.sh` to provide users with some context.

Reply:
I updated the pull request with a little more elaborate init script that has both dev and production mode and set extraArguments showing how that is done, is this close to what you had in mind?

Reply:
yes, thank you!